ci/update-checker/src/main/scala/com/twitter/intellij/updatechecker/VersionsConf.scala

```diff
@@ -0,0 +1,28 @@
+package com.twitter.intellij.updatechecker
+
+import com.typesafe.config.ConfigRenderOptions
+import org.virtuslab.ideprobe.ConfigFormat
+import org.virtuslab.ideprobe.dependencies.{IntelliJVersion, Plugin}
+import pureconfig.ConfigWriter
+import pureconfig.generic.semiauto.deriveWriter
+
+case class VersionsConf(intellij: IntelliJVersion, plugins: Map[String, Plugin])
+
+object VersionsConfFormat extends ConfigFormat {
+
+  private case class VersionsConfProtocol(versions: VersionsConf)
+
+  private val renderJson = ConfigRenderOptions
+    .defaults().setComments(false).setFormatted(true).setOriginComments(false)
```

Comment:
setJson(true) here for consistency, I will soon update it in release tool as well

Reply:
Done